### DISCUSSION TITLE
Mark Episodes as finished

### DISCUSSION THREAD

**Discussion by xTiLTEDx** [madeofpendletonwool/PinePods#163] [Ideas]
Hey madeofpendletonwool, 

can we get an optional Option mark Episodes as finished? I came from Google Podcast (wich will be killed next Month) and want to mark allready heard Episodes. 

Thank you for your Hard work and the nice App!

**Comment by madeofpendletonwool** [comment]
Excellent idea, and very easy to add on. Would you want the idea to manually mark episodes as listened or automatically mark episodes completed once they are finished? I assume probably a combination of the two?

**Reply by xTiLTEDx** [reply]
Yes a combination of this two. Maby a mark all option or mark several episodes. And also mark as finished when an Episode ends. Also it would by nice if the Played episodes greyed out and dont show in the Main Feed list, only in the Show List. This would be amazing.
